dist/origin-web-common-services.js

```diff
@@ -693,8 +693,23 @@ angular.module('openshiftCommonServices')
 
         // Handle web socket requests with a parameter
         if (config.method === 'WATCH') {
-          config.url = URI(config.url).addQuery({access_token: token}).toString();
-          authLogger.log('AuthService.addAuthToRequest(), added token param', config.url);
+          // Ensure protocols is defined
+          config.protocols = config.protocols || [];
+
+          // Ensure protocols is an array
+          if (!_.isArray(config.protocols)) {
+            config.protocols = [config.protocols];
+          }
+
+          // Ensure protocols has at least one item in it (for the server to echo back once the bearer protocol is stripped out)
+          if (config.protocols.length == 0) {
+            config.protocols.unshift("undefined");
```

Comment:
could we do something like console.openshift.io.placeholder or something?  I worry that seeing "undefined" in a request handshake might cause someone to open a bug/ticket